openslides/topics/static/js/topics/site.js

```diff
@@ -6,12 +6,17 @@ angular.module('OpenSlidesApp.topics.site', ['OpenSlidesApp.topics'])
 
 .config([
     '$stateProvider',
-    function($stateProvider) {
+    'gettext',
+    function($stateProvider, gettext) {
+        gettext('Topic');
         $stateProvider
             .state('topics', {
                 url: '/topics',
                 abstract: true,
                 template: "<ui-view/>",
+                data: {
+                    title: 'Topic',
```

Comment:
Use Topics here. I think @emanuelschuetze said to use Topic only because of missing translation. Now this problem should be solved.